### DISCUSSION TITLE
:)

### DISCUSSION THREAD

**Discussion by StarLordS7S** [Monika-After-Story/MonikaModDev#8784] [appreciation]
Just gonna say great fans, great devs, great ideas this mod is the best. :) I know lots have said this but good job on the b-day update I had a wonderful time. :) Everyone here is amazing, thank you for taking the time to make something for me, others, and yourselves even though you didn't have to do this you still did. :) I appreciate it. :) I am very curious as to what else you guys will do and i'm excited for the voice recognition thing looks cool. THANKS AGAIN AND TAKE CARE :) 
P.S.
cant stop with the smile emoji :)

**Comment by Sujeito17** [comment]
Yup... but ;-; is better

**Comment by OyasumiPunpun** [comment]
I agree. Even though the bday event was ruined because of a knife on the **wish**list that Monika didn't **wish** (the irony) this mod is still great.

**Comment by dk107605** [comment]
Keep up the good work, MAS devs!
